StreamChatClient_v3/Database/DatabaseContainer.swift

```diff
@@ -107,6 +107,32 @@ class DatabaseContainer: NSPersistentContainer {
             }
         }
     }
+    
+    /// Removes all data from the local storage.
+    ///
+    /// - Warning: ⚠️ This is a non-recoverable operation. All data will be lost after calling this method.
+    ///
+    /// - Parameters:
+    ///   - force: If sets to `false`, the method fails if there are unsynced data in to local storage, for example
+    /// messages pedning sent. You can use this option to warn a user about potential data loss.
+    ///   - completion: Called when the operation is completed. If the error is present, the operation failed.
+    func flush(force: Bool, completion: ((Error?) -> Void)? = nil) {
```

Comment:
> the current content of the DB is flushed (but the instance itself stays, so all existing controllers and workers are still alive and keep working)
I don't think we should do that. We discussed that user switch will switch to different database file source. So we can keep offline data for different users. Of course this is a rare operation from users, but why we should remove?
What's more important that user will connect on `setUser` with empty database if developers will do that because bad flow.

Reply:
Discussed in Slack